LondonBritishLibrary/orient/BLorient596.xml

```diff
@@ -0,0 +1,223 @@
+                        </msIdentifier>
+
+<msContents>
+
+  <msItem xml:id="ms_i1">
+  <locus from="1r"/>
+  <title xml:lang="en" type="complete" ref="LIT2509Weddas">Wǝddāse Māryām, or Encomium of the blessed Virgin Mary, arranged in portions
+  for the days of the week</title>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i2">
+  <locus from="19r"/>
+  <title xml:lang="en" type="complete" ref="LIT3108RepCh388">A hymn</title>
+  <incipit xml:lang="gez">ይዌድስዋ፡ መላእክት፡ ለማርያም፡ በውስተ፡ ውሣጤ፡ መንጦላዕት፡</incipit>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i3">
+  <locus from="21v"/>
+  <title xml:lang="en" type="complete">A prayer</title>
+  <incipit xml:lang="gez">ኦእግዚእትየ፡ ማርያም፡ ኦብፅዕት፡ ዘአልባቲ፡ ርስሐት፡</incipit>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i4">
+  <locus from="23r"/>
+  <title xml:lang="en" type="complete">Hymns and prayers</title>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i4.1">
+  <locus from="23r"/>
+  <title xml:lang="en" type="incomplete">To the blessed Virgin Mary</title>
+  <note>Imperfect at the beginning.</note>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i4.2">
+  <locus from="23vb"/>
+  <title xml:lang="en" type="complete" ref="LIT3088RepCh368">To the same</title>
+  <incipit xml:lang="gez">ኦርኅርኅተ፡ ሕሊና፡ አፍቅሮተ፡ ሰብእ፡ ልማዳ፡</incipit>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i4.3">
+  <locus from="28r"/>
+  <title xml:lang="en" type="complete" ref="LIT2839RepCh123">To our Lord</title>
+  <incipit xml:lang="gez">ሰላም፡ ለዝክረ፡ ስምከ፡ ስመ፡ መሐላ፡ ዘኢይሔሱ፡</incipit>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i4.4">
+  <locus from="38r"/>
+  <title xml:lang="en" type="complete" ref="LIT2883RepCh164">To the Saviour of the World <foreign xml:lang="gez">መድኃኔ፡ ዓለም፡</foreign></title>
+<incipit xml:lang="gez">ሰላም፡ ለዝክረ፡ ስምከ፡ ዘኢረከቡ፡ ተፍጻሜተ፡</incipit>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i4.5">
+  <locus from="46r"/>
+  <title xml:lang="en" type="complete" ref="LIT2835RepCh119">To S. Michael</title>
+  <incipit xml:lang="gez">ሰላም፡ ለዝክረ፡ ስምከ፡ ምስለ፡ ስመ፡ ልዑል፡ ዘተሳተፈ፡</incipit>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i4.6">
+  <locus from="54v"/>
+  <title xml:lang="en" type="complete" ref="LIT2965RepCh246">To the angel Gabriel</title>
+  <incipit xml:lang="gez">በሰላመ፡ ገብሬል፡ መልአክ፡ በላዕለ፡ ማርያም፡ ዘአዕረፈ፡</incipit>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i4.7">
+  <locus from="60r"/>
+  <title xml:lang="en" type="complete">A prayer to our Lord</title>
+  <incipit xml:lang="gez">ኦእግዚእየ፡ ኢ'፡ ክ'፡ ወልደ፡ እግዚአብሔር፡ ሕያው፡ ወወልደ፡ ማርያም፡ ሥግው፡ ዕቀበኒ፡ እምኵሉ፡ እኩይ፡</incipit>
+  <textLang mainLang="gez"/>
+  </msItem>
+
+  <msItem xml:id="ms_i4.8">
+  <locus from="71v"/>
+  <title>Another prayer</title>
+  <incipit xml:lang="gez">ኦእግዚእየ፡ ኢ'፡ ክ'፡ ወልደ፡ እግዚ'፡ ሕያው፡ በእንተ፡ ማርያም፡ ወላዲትከ፡ ርድአኒ፡</incipit>
```

Comment:
I suppose that Wright just omitted the text for brevity, I would either write it our or encode the omission using `<gap>`